lib/chat_api/repo.ex

```diff
@@ -1,7 +1,17 @@
 defmodule ChatApi.Repo do
+  @cursor_pagination_defaults [
+    maximum_limit: 50
+  ]
+
   use Ecto.Repo,
     otp_app: :chat_api,
     adapter: Ecto.Adapters.Postgres
 
   use Scrivener, page_size: 50
+
+  @spec paginate_with_cursor(any, keyword, any) :: Paginator.Page.t()
+  def paginate_with_cursor(queryable, opts \\ [], repo_opts \\ []) do
```

Comment:
https://hexdocs.pm/paginator/Paginator.html#module-use-without-macros
The `Scrivener` macro uses `Repo.paginate`, so i have to use `Paginator` without a macro, and use a different function name